doc/performance_blog_post/performance_blog_post.md

```diff
@@ -0,0 +1,120 @@
+
+## How it Begins
+
+For me this project started when our product manager asked me to improve the performance of the Parquet file loading. He compared the performance of our products with other solutions. As a result of this comparison, we realized that our Parquet loading needs major improvements.
+
+In order to do professional performance improvements we use the simple workflow:
+
+Measure &#8594; Improve &#8594; Measure.
+
+In my eyes, this simple strategy brings lots of advantages:
+
+First, it allows you to check whether a change helped or if it even made some functionality slower. For example improving the performance of loading big Parquet files can easily slow down the loading of many small files. Second, this strategy allows you to define a goal when to stop. Otherwise, you will keep improving forever since there's always a space for improvements.
+
+## Writing Performance Tests
+
+So my first step was to write automated performance tests for the Parquet loading. Don't get me wrong. It's not that we did not have any tests before. Of course, we had unit and integration tests for the Virtual Schema that make sure that it works as expected. But since this is a rather new product we did not have automated performance tests yet.
+
+Until now, we checked the performance of Virtual Schema manually by starting a cluster and querying a Parquet formatted dataset. However, for these performance improvements we needed more test cases and also more reliability.
+
+While for our integration tests we use the docker version of Exasol (which is absolutely sufficient for functional testing), for the performance test we want to test with a real-live setup. So we decided to test with a 4 node cluster (with `c5.4xlarge` instance types) on AWS. There, we load the files from S3 using the [s3-document-files-virtual-schema](https://github.com/exasol/s3-document-files-virtual-schema).
+
+The Virtual Schemas are built out of multiple layers:
+
+![Stack of virtual schema layers](vsDocStack.png)
+
+The Parquet loading implementation is located in the Document Files Virtual Schema. However, we can't test this directly. Instead, we decided to test the S3 implementation, since this is the most popular dialect.
+
+### Test Quality
+
+We measure the query performance by measuring the execution time of a query on a Virtual Schema table. By that, we measure the full execution time including network transmission, S3 loading and so on. This of course adds some noise to our measurement. However, we decided for this strategy since it measures the time that our customers can expect.
+
+In order to make the tests more reliable we run each test 5 times. Then we drop the first 2 results, since they are usually slower. Then we build the average of the remaining 3 runs. We decided for the average since it will again result in a value that customers can expect.
```

Comment:
```suggestion
In order to make the tests more reliable, we run each test 5 times. Then we drop the first 2 results since they are usually slower. Then we build the average of the remaining 3 runs. We decided on the average since it will again result in a value that customers can expect.
```